Header header
uint32 object_type
uint32 object_id
geometry_msgs/Pose pose
geometry_msgs/Point32 size
float32 support
